colossalai/booster/plugin/gemini_plugin.py

```diff
@@ -53,24 +54,27 @@ def load_unsharded_model(self, model: GeminiDDP, checkpoint: str, strict: bool =
         Load model from checkpoint with automatic unwrapping.
         The model should be unwrapped in self.load_model via ModelWrapper.unwrap.
         """
-        super().load_unsharded_model(model, checkpoint, strict=strict)
+        assert isinstance(model, GeminiDDP), "Please boost the model before loading!"
+        super().load_unsharded_model(model.unwrap(), checkpoint, strict=strict)
```

Comment:
I notice that `load_sharded_model` does not unwrap model. Why?

Reply:
sorry, I forgot to unwrap that, now modified.